lib/prism/desugar_compiler.rb

```diff
@@ -160,7 +160,7 @@ def desugar_operator_write_node(node, read_class, write_class, *arguments)
           0,
           read_class.new(*arguments, node.name_loc),
           nil,
-          node.operator_loc.slice.chomp("="),
+          node.operator_loc.slice.chomp("=").to_sym,
```

Comment:
This was caught by steep, and I made this change because `CallNode` uses a Symbol for `name`